tests/provision_amazon.sh

```diff
@@ -8,7 +8,7 @@ function rpm_install() {
     rm -f $package
 }
 
-release=$(awk -F \: '{print $5}' /etc/system-release-cpe)
+release=6
```

Comment:
Wouldnt this use EL6 repos on Amazon Linux 2 which should use EL7 repos?

Reply:
Yep, I need to fix that.